Demo/Application/Features/Fraud Protection - BTDataCollector/BraintreeDemoBTDataCollectorViewController.swift

```diff
@@ -0,0 +1,88 @@
+import UIKit
+import CoreLocation
+import BraintreeDataCollector
+import BraintreeCore
+
+class BraintreeDemoBTDataCollectorViewController: BraintreeDemoPaymentButtonBaseViewController {
+
+    let locationManager = CLLocationManager()
+    var dataLabel = UILabel()
```

Comment:
take or leave: Lazy instantiation would allow for `dataLabel` to be non-nil. The initial `UILabel` created here ends up being overwritten in `createPaymentButton()`.

Reply:
Using a `lazy var` would still require the property to have an initializer. Since we need access to this property in `tappedCollect` to update the text I think it makes sense as is to create an empty label them override it later.